cypress/integration/app_admin/app_admin.js

```diff
@@ -1,17 +1,15 @@
-import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
-
-const USER_NAME = 'test@unit.no';
-const APP_ADMIN_ROLE = 'App-admin';
+import { Given, When, Then, Before, After, And } from 'cypress-cucumber-preprocessor/steps';
+import { ADMIN_USER, ADMIN_NAME } from '../../support/constants';
 
 Before(() => {
-  cy.deleteUser(USER_NAME).then(() => {
-    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
-      cy.wrap(userId).as('userId');
+  cy.deleteUser(ADMIN_USER).then(() => {

```

Comment:
`createUser` starter med `deleteUser`, så den trengs sikkert ikke her også? (gjelder alle plasser hvor `createUser` kalles)

Reply:
Agh, ser det har blitt to metoder med samme navn, `cy.createUser` i command.js og `createUser `i users.js. Navner om `cy.createUser` til `cy.createCognitoUser` for å redusere forvirring.

Reply:
Ser jeg må rydde opp litt mer. Skal være konsistent i bruken av funksjonene.